package.json

```diff
@@ -84,14 +89,23 @@
       "stylelint-no-unsupported-browser-features"
     ],
     "rules": {
+      "string-quotes": "single",
+      "color-hex-length": "long",
```

Comment:
And if we're going to use a common stylelint ruleset, it's fair game for this beast to evolve in time, and add our own x-touch to it. 
Having Wordpress Standards as a starting point is perfectly fine.